http/pinger.go

```diff
@@ -1,18 +1,19 @@
-package websocket
+package http
 
```

Comment:
could we move this pinger to `http/websocket/` as it's applicable only to websocket. Does this make sense?

Reply:
Done